src/commands/sfpowerkit/source/picklist/generatepatch.ts

```diff
@@ -232,11 +232,14 @@ export default class Generatepatch extends SFPowerkitCommand {
             fs.copyFileSync(zipFile, `${dir}${packageToBeUsed.package}_picklist.zip`);
 
             //Store it to static resources
-            const metadata = `<?xml version="1.0" encoding="UTF-8"?>	
-      <StaticResource xmlns="http://soap.sforce.com/2006/04/metadata">	
-          <cacheControl>Public</cacheControl>	
-          <contentType>application/zip</contentType>	
-      </StaticResource>`;
+            const metadata = `
```

Comment:
Is this XML equivalent to the previous version? Incorrect indentations may cause parse errors